### PR TITLE
Cleanup of PR #74 (Implementation of private key authenticator)

### PR DESCRIPTION
**Pull request by zanechua**
Cleanup of https://github.com/GrahamCampbell/Laravel-GitHub/pull/74 as per @GrahamCampbell comments. Don't see anything further that needs to be done. Did not test this because mostly it's just refactoring for proper naming so that people can easily understand what they need

### CODE REVIEW

File: tests/Authenticators/PrivateKeyAuthenticatorTest.php [hunk 1]
```diff
@@ -33,7 +33,7 @@ public function testMakeSuccess()
 
     /**
      * @expectedException \InvalidArgumentException
-     * @expectedExceptionMessage You must provide a valid key file
+     * @expectedExceptionMessage You must inform a valid key file
```
Comment (GrahamCampbell) [inline tests/Authenticators/PrivateKeyAuthenticatorTest.php]: inform is the wrong word

Comment (zanechua) [reply tests/Authenticators/PrivateKeyAuthenticatorTest.php]: I know that inform is the wrong word but that's what the upstream provider is using. I actually changed it once to have tests fail. Hence I changed it back. 

I've already created an upstream PR.

https://github.com/lcobucci/jwt/pull/283

Comment (GrahamCampbell) [reply tests/Authenticators/PrivateKeyAuthenticatorTest.php]: Thanks. I've left a comment on your upstream PR.

**Comment by GrahamCampbell** [comment]
Merged in #88. :)

**Pull request closed without merging**